hangman.py

```diff
@@ -72,9 +61,13 @@ def get_guessed_word(secret_word, letters_guessed):
     returns: string, comprised of letters, underscores (_), and spaces that represents
       which letters in secret_word have been guessed so far.
     '''
-    # FILL IN YOUR CODE HERE AND DELETE "pass"
-    pass
-
+    word = []
+    for letter in secret_word:
+        if letter in letters_guessed:
+            word += f'{letter} '
```

Comment:
You could append here a letter only, but in the return you could call `' '.join(...)`, which will add spaces between values

Reply:
Done